### PR TITLE
feat(ENV-3856): improve data seeding documentation

### PR DESCRIPTION
**Pull request by uribrecher**
This PR Addresses feat(ENV-3856): improve data seeding documentation:
* Add postgres data seeding blueprint sample

### CODE REVIEW

File: getting-started/aws/data-seeding/postgres.yaml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        fsGroup: 1001
+        runAsUser: 1001
+      terminationGracePeriodSeconds: 30
+      volumes:
+        - name: persistent-postgres-database-0
+          persistentVolumeClaim:
+            claimName: pvc-10gb
+---
+apiVersion: batch/v1
+kind: Job
```
Comment (yarelm) [inline getting-started/aws/data-seeding/postgres.yaml:87]: The app needs to "dependsOn" this as well no? I wonder if it should be part of this example

Comment (uribrecher) [reply getting-started/aws/data-seeding/postgres.yaml:87]: what app? there is no app in this example

Comment (uribrecher) [reply getting-started/aws/data-seeding/postgres.yaml:87]: what app? there is no app in this example

**Comment by uribrecher** [comment]
> @uribrecher can you link how this example is being used in our docs?

sure, I linked it to ENV-3856
but here it is:
https://velocitytech.atlassian.net/wiki/spaces/E/pages/591003769/Data+seeding+getting+started+guide

**Comment by idankinory** [comment]
@uribrecher Thanks.
This appears to be a very comprehensive guide.
Personally, I think it's a bit much for a getting started guide; perhaps we should divide it into an intro section and a more in-depth section, but we'll leave that decision to our documentation master Jeff.
Overall, more documentation is preferable to no documentation:)
Approving the PR.

**Review by idankinory — APPROVED** [review]

**Pull request merged**